### PR TITLE
Add XML comments to all public classes

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

File: src/neo-vm/Debugger.cs [hunk 1]
```diff
@@ -41,6 +57,12 @@ private void ExecuteAndCheckBreakPoints()
             }
         }
 
+        /// <summary>
+        /// Removes the breakpoint at the specified position in the specified script.
+        /// </summary>
+        /// <param name="script">The script to remove the breakpoint.</param>
+        /// <param name="position">The position of the breakpoint in the script.</param>
+        /// <returns></returns>
```
Comment (shargon) [inline src/neo-vm/Debugger.cs:65]:
```suggestion
        /// <returns>Returns true if the breakpoint was removed</returns>
```

File: src/neo-vm/Debugger.cs [hunk 2]
```diff
@@ -2,16 +2,28 @@
...
+    /// </summary>
     public class Debugger
     {
         private readonly ExecutionEngine engine;
         private readonly Dictionary<Script, HashSet<uint>> break_points = new Dictionary<Script, HashSet<uint>>();
 
+        /// <summary>
+        /// Create a debugger on the specified <see cref="ExecutionEngine"/>.
+        /// </summary>
+        /// <param name="engine"></param>
```
Comment (shargon) [inline src/neo-vm/Debugger.cs:16]:
```suggestion
        /// <param name="engine">The ExecutionEngine to debug</param>
```

File: src/neo-vm/Debugger.cs (hunk 1 above)
Comment (erikzhang) [reply src/neo-vm/Debugger.cs]: Updated.

File: src/neo-vm/Debugger.cs (hunk 2 above)
Comment (erikzhang) [reply src/neo-vm/Debugger.cs]: Updated.

**Review by shargon — APPROVED** [review]

**Pull request merged**